orchagent/fgnhgorch.h

```diff
@@ -99,10 +108,16 @@ class FgNhgOrch : public Orch, public Observer
     NeighOrch *m_neighOrch;
     IntfsOrch *m_intfsOrch;
     VRFOrch *m_vrfOrch;
+
     FgNhgs m_FgNhgs;
     FGRouteTables m_syncdFGRouteTables;
+    FgNhgMembers m_fgNhgMembers;
```

Comment:
I think for clarity, is it better to call this `m_fgNhgNexthops`?. Members is generic that it can also include prefixes

Reply:
Thanks, will do so.